### PR TITLE
UI for energy restore items

### PR DESCRIPTION
**Pull request by Ishadijcks**
Closes #327 

Based on @fuws works in #327 

image

### CODE REVIEW

**Comment by RedSparr0w** [comment]
Nice, do you think it look better on the side?

**Comment by RedSparr0w** [comment]
Maybe similar to this?:
<details>
<summary>html code</summary>

```html
                       <table width="100%" style="margin-bottom: 10px;">
                            <tbody><tr>
                                <td width="33.33%">
                                    <button style="display: inline;" class="btn btn-secondary">
                                        <img class="restore-item" src="assets/images/items/SmallRestore.png">
                                        <span>&nbsp;+3 energy</span>
                                        <span>(5)</span>
                                    </button>
                                </td>
                                <td width="33.33%">
                                    <button style="display: inline;" class="btn btn-secondary">
                                        <img class="restore-item" src="assets/images/items/MediumRestore.png">
                                        <span>&nbsp;+6 energy</span>
                                        <span>(1)</span>
                                    </button>
                                </td>
                                <td width="33.33%">
                                    <button style="display: inline;" class="btn btn-secondary">
                                        <img class="restore-item" src="assets/images/items/LargeRestore.png">
                                        <span>&nbsp;+15 energy</span>
                                        <span>(30)</span>
                                    </button>
                                </td>
                            </tr>
                        </tbody></table>
```
</details>

Screenshot from 2019-06-02 10-51-30

**Comment by Ishadijcks** [comment]
image

**Comment by Ishadijcks** [comment]
image

**Comment by RedSparr0w** [comment]
Another couple changes that i think would be good (not really related to this PR but hey):
- Enlarge the progress bar
- Show current energy in the progress bar
- spread the bottom buttons accross the entire area

<details>
<summary>html code for progress bar</summary>

```html
<div class="row" style="text-align:center;">
                       <div class="progress" style="
    width: calc(32px * 25);
">
                           <div id="mineEnergyBar" class="progress-bar" role="progressbar" aria-valuemin="0" aria-valuemax="100" data-bind="style: { width: player._mineEnergy()/player._maxMineEnergy()*100 + '%' }" style="width: 100%;height: 26px;font-size: 20px;"><span style="
    top: 3px;
    position: relative;
">50/50 <img src="assets/images/underground/flash.png" style="
    filter: invert(100%);
    "> (next 33s)</span></div>
                       </div>
                       
                   </div>
```

</details>

Screenshot from 2019-06-02 11-03-48

File: src/components/underground.html [hunk 1]
```diff
@@ -19,6 +19,43 @@
        <div class="tab-content">
            <div id="dig" class="tab-pane fade in active show">
                <div class="modal-body" style="width: 80%;margin: auto;">
+
+                   <table class='table table-sm' style="width:60%; margin: auto">
+                       <tbody>
+                           <tr>
+                               <td class="col-md-4">
```
Comment (Aegyo) [inline src/components/underground.html]: Browsers are smart enough to close these `<td>` elements for us, but it would be nice if we did it.

**Pull request merged**